components/notification/NotificationPaginator.vue

```diff
@@ -13,6 +13,14 @@ const virtualScroller = false // TODO: fix flickering issue with virtual scroll
 
 const groupCapacity = Number.MAX_VALUE // No limit
 
+function includeNotificationsForStatusCard({ type, status }: mastodon.v1.Notification) {
+  // Exclude update, mention, pool and status notifications without the status entry:
+  // no makes sense to include them
+  // Those notifications will be shown using StatusCard SFC:
+  // check NotificationCard SFC L68 and L81 => :status="notification.status!"
+  return !((type === 'update' || type === 'mention' || type === 'poll' || type === 'status') && !status)
```

Comment:
maybe more clear if

```suggestion
  return status && !['update', 'mention', 'poll', 'status'].includes(type)
```

Reply:
Status is not required in some types

Reply:
@ayoayco you suggestion is wrong, should be `return status || ![...`

Reply:
done